vector/src/main/res/layout/item_splash_carousel.xml

```diff
@@ -39,7 +39,7 @@
 
         <ImageView
             android:id="@+id/carousel_item_image"
-            android:layout_width="wrap_content"
+            android:layout_width="match_parent"
```

Comment:
now that we have consistent bounding boxes we can matching the parent (which is constrained by gutters)